.gitignore

```
@@ -0,0 +1 @@
+src/api-config/api.js
```

Навіщо ігрнорити цей файл, якщо змінні можна тримати в файлах `.env`? Це best practice.

Також було б добре додати `.idea` та файли з MacOS в ігнор

```suggestion
.env
.idea
.DS_Store
```